accounts/abi/abi.go

```diff
@@ -255,7 +256,7 @@ func UnpackRevert(data []byte) (string, error) {
 		return "", errors.New("invalid data for unpacking")
 	}
 	if !bytes.Equal(data[:4], revertSelector) {
-		return "", errors.New("invalid data for unpacking")
+		return hexutils.BytesToHex(data), nil
```

Comment:
No need to reach into keycard-go to convert from bytes to hexstring. You could use e.g. `fmt.Sprintf("#%x", data)`

Reply:
done